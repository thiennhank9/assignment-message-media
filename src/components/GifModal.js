import Modal from "react-bootstrap/Modal";

export default function GifModal(props) {
  const { src, name, ...restProps } = props;

  return (
    <Modal centered size="lg" {...restProps}>
      <Modal.Header closeButton>
        <Modal.Title>Gif</Modal.Title>
      </Modal.Header>
      <Modal.Body>
        <img src={src} alt={name} className="w-100" />
      </Modal.Body>
    </Modal>
  );
}
