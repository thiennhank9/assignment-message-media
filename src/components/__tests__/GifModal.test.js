import renderer from "react-test-renderer";
import GifModal from "../GifModal";

test("GifModal - render correctly", () => {
  const tree = renderer.create(<GifModal />).toJSON();
  expect(tree).toMatchSnapshot();
});
