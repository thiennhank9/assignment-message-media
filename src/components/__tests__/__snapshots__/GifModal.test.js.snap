// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifModal - render correctly 1`] = `null`;
